pyInterface/package/_amplitude.py

```diff
@@ -12,14 +12,12 @@ def calcAmplitude(inputFileName,
                   maxNumberOfEvents = -1,
                   printProgress = True):
 
-	printDebug = pyRootPwa.utils.printDebug
 	printInfo = pyRootPwa.utils.printInfo
 	printSucc = pyRootPwa.utils.printSucc
 	printWarn = pyRootPwa.utils.printWarn
-	printErr = pyRootPwa.utils.printErr
 
 	printInfo("Calculating amplitude for key file '" + keyFileName + "' (index " + str(waveDescriptionID) +
-	                          ") with input file '" + inputFileName + "', output file '" + outputFileName + "'.")
+              ") with input file '" + inputFileName + "', output file '" + outputFileName + "'.")
```

Comment:
Please aligne the two `"` at the beginning. Also, the line should start with one tab.